geth/api/backend.go

```diff
@@ -423,3 +434,10 @@ func (b *StatusBackend) NotifyUsers(message string, payload fcmlib.NotificationP
 
 	return err
 }
+
+func appendIf(condition bool, services []gethnode.ServiceConstructor, service gethnode.ServiceConstructor) []gethnode.ServiceConstructor {
```

Comment:
Not sure if creating a new function worth just to shave off 2 lines of `if {}` block one time.

Reply:
The idea is to remove all `activateXYZService` functions and move the invocation here. So it will look like this:
```go
appendIf(conditionA, serviceConstructorA)
appendIf(conditionB, serviceConstructorB)
// ...
```
It should have more sense in this context.